literature-cleaned.bib

```diff
@@ -55,6 +65,14 @@ @inproceedings{KTS:FormaliSE21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@thesis{A21,

```

Comment:
We use `@mastersthesis` here. Is `@thesis` a valid tag? If so, do we want to change `@mastersthesis` for all bachelor theses to `@thesis`?

Reply:
I changed it to `@mastersthesis` for now.